### PR TITLE
Abonas fixcomparingprbymergedatfield

### PR DESCRIPTION
**Pull request by jampunkramadhan**
git log --pretty=oneline
git push origin --tags

### CODE REVIEW

**Comment by jampunkramadhan** [comment]
@

File: CHANGELOG.md [hunk 1]
```diff
@@ -245,6 +256,7 @@
 - Add a rake task [\#260](https://github.com/skywinder/github-changelog-generator/pull/260) ([raphink](https://github.com/raphink))
 - Add release\_url option [\#259](https://github.com/skywinder/github-changelog-generator/pull/259) ([raphink](https://github.com/raphink))
 - Add --since-tag [\#257](https://github.com/skywinder/github-changelog-generator/pull/257) ([raphink](https://github.com/raphink))
+>>>>>>> refs/remotes/skywinder/master
```
Comment (olleolleolle) [inline CHANGELOG.md]: These changes could be outside the PR.

**Comment by jampunkramadhan** [comment]
diff --git a/CHANGELOG.md b/CHANGELOG.md
index be8c8ff..442a000 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,16 @@
 # Change Log
 
+<<<<<<< HEAD
+## [Unreleased](https://github.com/skywinder/github-changelog-generator/tree/HEAD)
+
+[Full Changelog](https://github.com/skywinder/github-changelog-generator/compare/1.7.0...HEAD)
+
+**Merged pull requests:**
+
+- Add a rake task [\#260](https://github.com/skywinder/github-changelog-generator/pull/260) ([raphink](https://github.com/raphink))
+- Add release\_url option [\#259](https://github.com/skywinder/github-changelog-generator/pull/259) ([raphink](https://github.com/raphink))
+- Add --since-tag, close  [\#257](https://github.com/skywinder/github-changelog-generator/pull/257) ([raphink](https://github.com/raphink))
+=======
 ## [1.12.0](https://github.com/skywinder/github-changelog-generator/tree/1.12.0) (2016-04-01)
 [Full Changelog](https://github.com/skywinder/github-changelog-generator/compare/1.11.8...1.12.0)
 
@@ -245,6 +256,7 @@
 - Add a rake task [\#260](https://github.com/skywinder/github-changelog-generator/pull/260) ([raphink](https://github.com/raphink))
 - Add release\_url option [\#259](https://github.com/skywinder/github-changelog-generator/pull/259) ([raphink](https://github.com/raphink))
 - Add --since-tag [\#257](https://github.com/skywinder/github-changelog-generator/pull/257) ([raphink](https://github.com/raphink))
+>>>>>>> refs/remotes/skywinder/master
 
 ## [1.7.0](https://github.com/skywinder/github-changelog-generator/tree/1.7.0) (2015-07-16)
 [Full Changelog](https://github.com/skywinder/github-changelog-generator/compare/1.6.2...1.7.0)
diff --git a/lib/github_changelog_generator/generator/generator_generation.rb b/lib/github_changelog_generator/generator/generator_generation.rb
index b22fe40..efaa14d 100644
--- a/lib/github_changelog_generator/generator/generator_generation.rb
+++ b/lib/github_changelog_generator/generator/generator_generation.rb
@@ -118,7 +118,7 @@ def generate_log_between_tags(older_tag, newer_tag)
     #
     # @return [Array] filtered issues and pull requests
     def filter_issues_for_tags(newer_tag, older_tag)
-      filtered_pull_requests = delete_by_time(@pull_requests, :actual_date, older_tag, newer_tag)
+      filtered_pull_requests = delete_by_time(@pull_requests, :merged_at, older_tag, newer_tag)
       filtered_issues = delete_by_time(@issues, :actual_date, older_tag, newer_tag)
 
       newer_tag_name = newer_tag.nil? ? nil : newer_tag["name"]

**Comment by olleolleolle** [comment]
I'll close this - looks like robot spam.

**Pull request closed without merging**